packages/@okta/vuepress-site/docs/concepts/event-hooks/index.md

```diff
@@ -100,7 +98,7 @@ The `debugData` section of the [LogEvent](/docs/reference/api/system-log/#exampl
 
 Note that this information is available in the event regardless of whether the delivery was successful or failed.
 
-Thus, in conjunction with the `event_hook.delivery` System Log event, you can debug an end-to-end flow.
+Thus, in conjunction with the [System Log event](docs/reference/api/event-types/?q=event_hook.delivery) for event hook delivery failures, you can debug an end-to-end flow.
```

Comment:
1. This link needs to start with `/`, so `/docs/reference/...`
2. `event_hook.delivery` doesn't appear to be a valid event at this time.

Reply:
ack @ (1), will fix 
for (2) - That event was removed from Beta tag last week. I believe the new events catalog will be published this week as part of the major release?